frontend/apps/web/cypress/integration/Subsystem.spec.ts

```diff
@@ -0,0 +1,57 @@
+describe('Subsystem specs', () => {
+  it('Should check subsystem page', () => {
+    //visit home page
+    cy.visit('/')
+    //get first system link and click on it
+    cy.get('tbody>tr>td>div>span>a').eq(0).click()
```

Comment:
lets createa a folder "selectors" and there add table.ts
and add something like this
```
const Table = {
    firstRow: () => {
        return cy.get('tbody > tr:first-child > td > div > span > a');
    },
};

export { Table };

```

ili samo
```
 /**
     * All selectors related to Table component
     */
const Table = {
     /**
     * All links in a table
     */
    allDirectLinks: () => {
        return cy.get('tbody > tr> td > div > span > a');
    },
};

export { Table };

```

Reply:
Then you can use it like this:

```javascript
import { WizardNavigation } from '../../selectors/wizardNavigation';
context('Test System page ', () => {
   

    it('Selects first link in a table and  clicks on it (or somthing) and checks if that is done', () => {
        Table.firstDirectLink().click();
        //check if you are redirected to that subsystem
    });

```

point of some test could be, Find a link with a text "chain" click on it and see if you changed the link to system/chain.